### PR TITLE
Update wordpress-dotorg-plugin-deploy.md

### PR DESCRIPTION
**Pull request by jeffpaul**
Something's a bit off with the linkages here, the GH Stars display isn't working on https://github-actions.explore-tech.org/wordpress-dotorg-plugin-deploy, so trying a tweak here to the GH URL to see if that fixes things.

### CODE REVIEW

**Comment by boyney123** [comment]
@jeffpaul cool.

Here is a link with your new changes: https://deploy-preview-43--github-actions.netlify.com/

Can you confirm everything is cool? If it is I will merge 👍

**Comment by jeffpaul** [comment]
@boyney123 blerg, that doesn't look resolved.  Any idea why those GH Stars aren't showing?

File: src/actions/wordpress-dotorg-plugin-deploy.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 path: "/wordpress-dotorg-plugin-deploy"
 title: "WordPress.org Plugin Deploy"
-github_url: "https://github.com/10up/actions-wordpress/dotorg-plugin-deploy"
+github_url: "https://github.com/10up/actions-wordpress/tree/develop/dotorg-plugin-deploy"
```
Comment (boyney123) [inline src/actions/wordpress-dotorg-plugin-deploy.md]:
```suggestion
github_url: "https://github.com/10up/actions-wordpress"
```
We might need to move this to this url, for the stars to work, did you want to try that?

**Review by boyney123 — APPROVED** [review]
Hey @jeffpaul 👋 

Looks like the url might have to change to the root Github page, I have made a suggestion, did you want to give that a go?

**Comment by jeffpaul** [comment]
@boyney123 will be interesting to see how that results as there are now two actions in that repo, 🤞 .

**Comment by jeffpaul** [comment]
@boyney123 we split our actions out into individual repos for each, so this updates the link accordingly.  Hopefully that fixes the display on your site.

**Comment by boyney123** [comment]
Thanks again @jeffpaul

**Pull request merged**